apps/web/src/components/ScreeningDecisions/ScreeningDecisionDialog.tsx

```diff
@@ -296,7 +296,9 @@ export const ScreeningDecisionDialog = ({
         ? getAssessmentStepType(assessmentStep?.type)
         : commonMessages.notApplicable,
     ),
-    customTitle: getLocalizedName(assessmentStep?.title, intl),
+    customTitle: assessmentStep.title
+      ? getLocalizedName(assessmentStep?.title, intl)
+      : "",
```

Comment:
You should be able to just pass `true` as the third arg to `getLocalizedName` to do this exact same thing.

https://github.com/GCTC-NTGC/gc-digital-talent/blob/865d0ad5fad670c6fdccc7a998773ae69009a47f/packages/i18n/src/utils/localize.ts#L65-L69